adm_program/modules/preferences/preferences.php

```diff
@@ -513,6 +513,24 @@ function getPreferencePanel($group, $id, $parentId, $title, $icon, $body)
     $formValues['mail_sendmail_name'],
     array('maxLength' => 50, 'helpTextIdInline' => 'SYS_SENDER_NAME_DESC')
 );
+
+// Add js to show or hide mail options
+$page->addJavascript('
```

Comment:
I am not sure if this is the best place to add it or if there is any "default" js how to handle things like this.

Reply:
until now we have no default to handle things like this. But a default could be useful :-)